runtime/tutor

```diff
@@ -663,15 +667,16 @@ _________________________________________________________________
 =================================================================
 
  Type . to repeat the last insert command.
- Type A-. to repeat the last f / t selection.
+ Type Alt-. to repeat the last f / t selection.
 
  1. Move the cursor to the line marked '-->' below.
  2. Make a change, insertion or appendage and repeat it with . .
- 3. Try using A-. with f and t, to select multiple sentences for
-    instance.
+ 3. Try using Alt-. with f and t, to select multiple sentences
+    for instance.
 
  --> This is some text for you to repeat things. You can repeat
-     insertions like changing words, or repeat selections like f/t.
+     insertions like changing words, or repeat selections like
+     f / t.
 
```

Comment:
Make sure to delete blank lines if you add more lines of content, to keep the section spacing constant.
```suggestion
     insertions like changing words, or repeat selections like
     f / t.
```